README.md

```diff
@@ -17,25 +17,20 @@ You need a working installation of [NodeJS](http://nodejs.org/) and
 [zombie.js](http://zombie.labnotes.org) library through npm:
 
 ``` bash
-npm install -g zombie
+$ npm install -g zombie
 ```
 
 The driver requires zombie.js __version 2.0.0 or higher__.
 
 Use [Composer](https://getcomposer.org/) to install all required PHP dependencies:
 
-```json
-{
-    "require": {
-        "behat/mink":               "~1.6",
-        "behat/mink-zombie-driver": "~1.2"
-    }
-}
+```bash
+$ curl -sS https://getcomposer.org/installer | php
+$ php composer.phar install
 ```
 
 ```bash
-$> curl -sS https://getcomposer.org/installer | php
-$> php composer.phar install
+$ composer require --dev behat/mink behat/mink-zombie-driver
```

Comment:
> shouldn't it go in dev?

From one point of view Mink & it's drivers are dev dependency to the project, because they are used only for testing and not in production. From other point of view we don't know really, but we know that people use them and therefore there should go to `require`. @stof , which one you preffer?

>  People often forget to update the docs when versions change, so this way, they don't have to be constantly updated.

Question remains: what versions will be used in that case?